src/decoders/mod.rs

```diff
@@ -186,12 +186,43 @@ impl Camera {
   }
 }
 
-pub fn ok_image(camera: &Camera, width: usize, height: usize, wb_coeffs: [f32;4], image: Vec<u16>) -> Result<RawImage,String> {
-  Ok(RawImage::new(camera, width, height, wb_coeffs, image))
+/// Possible values for the IFD tag Orientation (0x0112)
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum Orientation {
```

Comment:
Yeah, I'm sure TIFF allows all the options. I doubt any camera uses them but it doesn't matter really.